.github/workflows/ci.yml

```
@@ -0,0 +1,38 @@
+name: CI Python 3 on Linux
+
+on: [push, pull_request]
```

Try:
```suggestion
on:
  push:
  pull_request:
  # Run daily at 0:01 UTC
  schedule:
  - cron:  '1 0 * * *'
```
(I'm not sure about the push/pr events, you may need to transform them to smth like `push: true` or `push: {}` so please do some experiments with this)

---

@matthewfeickert ^

---

Then use `if: github.event_name == 'scheduled'` on that heavy task.